### PR TITLE
introduced errors making a new chat

### PR DESCRIPTION
**Pull request by zanderz**
in https://github.com/keybase/client/pull/9331 - confusing between new and existing when picking users.

EDITED: works by no longer displaying pending conversations, should be good to go

@keybase/react-hackers

### CODE REVIEW

**Comment by zanderz** [comment]
when the list of selected users would make a convo that doesn't yet exist? It does.

**Comment by zanderz** [comment]
although it shows up in the list separate from "New Conversation", not highlighted, while you are picking users.

**Comment by zanderz** [comment]
IOW, by "not painted" I just meant it's not shown as selected

**Comment by zanderz** [comment]
ok pending conversations no longer show in the list and nothing seemed to blow up!

File: shared/actions/chat/manage-thread.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ function* _startConversation(action: ChatGen.StartConversationPayload): Saga.Sag
   if (forceImmediate && existing) {
     const newID = yield Saga.call(Shared.startNewConversation, existing.get('conversationIDKey'))
     yield Saga.put(ChatGen.createSelectConversation({conversationIDKey: newID}))
-  } else if (existing && !temporary) {
+  } else if (existing) {
```
Comment (akalin-keybase) [inline shared/actions/chat/manage-thread.js]: If we don't have pending conversations anymore, is the else case below still needed? Also maybe can get rid of `createAddPending` and the `temporary` param in the payload

File: shared/chat/inbox/index.js.flow [hunk 2]
```diff
@@ -59,6 +59,7 @@ export type Props = {
   smallTeamsHiddenBadgeCount: number,
   smallTeamsHiddenRowCount: number,
   toggleSmallTeamsExpanded: () => void,
+  inSearch: boolean,
```
Comment (akalin-keybase) [inline shared/chat/inbox/index.js.flow]: I think you can now remove `showNewConversation`

**Comment by zanderz** [comment]
@akalin After removing all that stuff, the header crashes because it still expects to find a pending conversation as currently selected, even if it isn't displayed that way in the list. I would prefer to merge this and attempt this removal again separately, what do you think?

**Comment by akalin-keybase** [comment]
Sure, sgtm!

**Comment by zanderz** [comment]
Thanks for the reviews today, Fred!

**Pull request merged**